chall10/edramire.py

```diff
@@ -0,0 +1,110 @@
+#! /usr/bin/env python3
```

Comment:
Be careful with no specificating >= `python3.5` in the shebang when you use features specific from python > 3.5, like `f-strings`.